src/utils/helpers.ts

```diff
@@ -1,3 +1,5 @@
 import { UserRole } from 'types';
 
 export const isNewbie = (role: UserRole) => role === UserRole.Newbie;
```

Comment:
pls add types for results of this helper functions: 
```suggestion
export const isNewbie = (role: UserRole): boolean => role === UserRole.Newbie;
```